src/models/git/FileStatus.js

```diff
@@ -0,0 +1,111 @@
+        break;
+      case '003': // The HEAD status is absent, the WORKDIR status is absent and the STAGE status is different from WORKDIR.
+        this.isUntracked = true;
+        break;
+      case '020': // The HEAD status is absent, the WORKDIR status is different from HEAD and the STAGE status is absent.
+        this.isNew = true;
+        this.isUntracked = true;
+        break;
+      case '022': // The HEAD status is absent, the WORKDIR status is different from HEAD and the STAGE status is identical to WORKDIR.
+        this.isAdded = true;
+        this.isStaged = true;
+        break;
+      case '023': // The HEAD status is absent, the WORKDIR status is different from HEAD and the STAGE status is different from WORKDIR.
+        this.isAdded = true;
+        this.isStaged = true;
+        this.hasUnstagedChanged = true;
+        break;
+      case '100': // The HEAD status is present, the WORKDIR status is absent and the STAGE status is absent.
+        this.isDeleted = true;
+        this.isStaged = true;
+        break;
+      case '101': // The HEAD status is present, the WORKDIR status is absent and the STAGE status is identical to HEAD.
+        this.isDeleted = true;
+        this.isUnstaged = true;
+        break;
+      case '103': // The HEAD status is present, the WORKDIR status is absent and the STAGE status is different from WORKDIR.
+        this.hasUnstagedChanged = true;
+        break;
+      case '110': // The HEAD status is present, the WORKDIR status is identical to HEAD and the STAGE status is absent.
+        this.isUnstaged = true;
+        break;
+      case '111': // The HEAD status is present, the WORKDIR status is identical to HEAD and the STAGE status is identical to HEAD.
+        this.isUnmodified = true;
+        break;
+      case '113': // The HEAD status is present, the WORKDIR status is identical to HEAD and the STAGE status is different from WORKDIR.
+        this.hasUnstagedChanged = true;
+        break;
+      case '120': // The HEAD status is present, the WORKDIR status is different from HEAD and the STAGE status is absent.
+        this.isUnstaged = true;
+        break;
+      case '121': // The HEAD status is present, the WORKDIR status is different from HEAD and the STAGE status is identical to HEAD.
+        this.hasUnstagedChanged = true;
+        this.isUnstaged = true;
+        break;
+      case '122': // The HEAD status is present, the WORKDIR status is different from HEAD and the STAGE status is identical to WORKDIR.
+        this.isStaged = true;
+        this.hasUnstagedChanged = true;
+        break;
+      case '123': // The HEAD status is present, the WORKDIR status is different from HEAD and the STAGE status is different from WORKDIR.
+        this.isStaged = true;
+        this.hasUnstagedChanged = true;
+        break;
+      default: break;
+    }
+  }
+
+  /**
+   * Get status color of file.
+   * @return {String} Name of color.
+   */
+  get color() {
```

Comment:
does not return color but a "color name" (=status)

Reply:
done

Reply:
what was done ? no modification tracked.